code/modules/reagents/reagents.dm

```diff
@@ -147,26 +149,32 @@
 	if(volume && holder)
 		remove_self(removed)
 
-/datum/reagent/proc/affect_blood(var/mob/living/carbon/M, var/alien, var/effect_multiplier)
-	return
+/datum/reagent/proc/apply_sanity_effect(mob/living/carbon/H, effect_multiplier)
```

Comment:
mob/living/carbon/human/H

Reply:
sure. i'll make other changes to support it too

Reply:
done